tests/test_models_circuit_features.py

```diff
@@ -314,3 +318,34 @@ def test_repeated_execute_with_noise(backend):
     target_state = np.stack(target_state)
     final_state = [backend.to_numpy(x) for x in final_state]
     backend.assert_allclose(final_state, target_state)
+
+
+def test_repeated_execute_probs_and_freqs(backend):
+    circuit = Circuit(1)
```

Comment:
Maybe would be good to extend this test to multiqubit circuit, to catch the issue reported by @wilkensJ.